code/api/src/modules/user/mutations.js

```diff
@@ -37,4 +37,56 @@ export const userRemove = {
     }
   },
   resolve: remove
-}
\ No newline at end of file
+<<<<<<< HEAD
+}
+
+// Update Email
+export const updateEmail = {
+  type: UserType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+    email: {
+      email: 'email',
+      type: GraphQLString
+    }
+  },
+  resolve: updateEmailResolver
+}
+
+// Update Address
+export const updateAddress = {
+  type: UserType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+    address: {
+      address: 'address',
+      type: GraphQLString
+    }
+  },
+  resolve: updateAddressResolver
+}
+
+// Upload Image
+export const updateImage = {
+  type: UserType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+    image: {
+      image: 'image',
+      type: GraphQLString
+    }
+  },
+  resolve: updateImageResolver
+}
+=======
+}
+>>>>>>> main
```

Comment:
`HEAD` and `main` here are a little worrisome and often indicate future errors 🧐